experiment_code/hackrl/experiment.py

```diff
@@ -804,6 +903,9 @@ def compute_gradients(data, learner_state, stats):
 
         stats["kickstarting_loss_bc"] += kickstarting_loss_bc.item()
         stats["kickstarting_coeff_bc"] += FLAGS.kickstarting_loss_bc
+    if FLAGS.use_ewc:
+        ewc_penalty = EWC_INSTANCE.penalty(model)
+        total_loss += ewc_penalty * FLAGS.ewc_penalty_scaler
```

Comment:
It would be great if we logged ewc_penalty similar to other auxiliary losses. Please have a look at `if FLAGS.use_kickstaring` for an example.